_posts/2021-11-29-when-estimating-a-software-project-you-should-not.md

```diff
@@ -0,0 +1,34 @@
+---
+title: When estimating a software project you should not...
+---
+
+I'm currently working on an internal GitHub project where, for a change, my estimations have been wildly off. Before this project, I'd had a good multi-year streak of never overshooting my estimates and consistently hitting deadlines. This break down, combined with some recent reading, has made me think about the myths we tell ourselves about software estimation so I figure I'd repeat some here. These are in a rough order from "most obvious" to "least obvious".
```

Comment:
I'd love to hear why your estimations have been wildly off...

Reply:
> I'd love to hear why your estimations have been wildly off...

Reply:
Good point. Elaborated in first paragraph.